README.md

```diff
@@ -2,6 +2,8 @@
 
 The p8e-cee-api allows for operations against the encrypted object store, with included support for multi-store replication, and creating and broadcasting scoped transmissions to the Provenance Blockchain.
 
+`p8e-cee-api` stands for the Provenance Contract Execution Environment API.
+
```

Comment:
Thoughts on flipping order of these two lines?

Reply:
done!